lib/v2/oup/index.js

```diff
@@ -0,0 +1,53 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+const rootUrl = 'https://academic.oup.com';
+
+module.exports = async (ctx) => {
+    const name = ctx.params.name;
+    const url = `${rootUrl}/${name}/issue`;
+
+    const response = await got(url);
+    const cookies = response.headers['set-cookie'].map((item) => item.split(';')[0]).join(';');
+    const $ = cheerio.load(response.data);
+    const list = $('div.al-article-items')
+        .map((_, item) => ({
+            title: $(item).find('a.at-articleLink').text(),
+            link: new URL($(item).find('a.at-articleLink').attr('href'), rootUrl).href,
+        }))
+        .get();
```

Comment:
Consider using the `limit` query parameter.

Reply:
Have not any good happens after add limit.